LEGO1/mxvideopresenter.cpp

```diff
@@ -135,7 +135,7 @@ MxVideoPresenter::AlphaMask::AlphaMask(const MxBitmap &p_bitmap)
     for (MxS32 i = 0; i < m_width; i++) {
       if (*t_ptr) {
         // TODO: Second CDQ instruction for abs() should not be there.
-        MxU32 shift = abs(offset) & 7;
+        MxS32 shift = abs(offset) & 7;
```

Comment:
This seems to be because of conflicting overloads since `VEC.H` pulls in `math.h` and there is another `abs` defined in `stdlib.h`. We should probably fix this as otherwise it won't be possible anymore to call `abs` on an unsigned integer in cases like this. Let me think what would be best here.

Reply:
I guess the easiest would just be to cast `shift` to `MxS32` in the next line (the call to `abs`) to resolve the conflict. That's what the compiler should have done implicitly before the introduction of `math.h` anyway.